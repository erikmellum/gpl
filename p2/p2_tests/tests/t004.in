foo[] = {1,2,3};
nums[] = {1.1, 0.2, .3};
strings[] = {hello, goodbye, compiler};
