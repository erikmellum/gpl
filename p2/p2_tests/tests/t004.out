parser.cpp::main() reading file tests/t004.in
parser.cpp::main() about to call yyparse()
type zero or more array declarations followed by ^D:

foo[] = {1, 2, 3};
nums[] = {1.1, 0.2, 0.3};
strings[] = {hello, goodbye, compiler};

parser.cpp::main() after call to yyparse()

no errors found by yyparse()

parser.cpp::main() end of program.
